packages/hawtio/src/ui/login/HawtioLogin.tsx

```diff
@@ -58,7 +58,7 @@ export const HawtioLogin: React.FunctionComponent = () => {
 
   return (
     <LoginPage
-      backgroundImgSrc={backgroundImages}
+      backgroundImgSrc={backgroundImages.lg as string}
```

Comment:
I don't like such adhoc type casting. do we really need this or shouldn't we change the code in a different way?

Reply:
Image sources now required strings not BackgroundImageSrcMap's

Reply:
The point is that we shouldn't use `as string` in an adhoc way. Why don't you change the definition of `backgroundImages` then?
https://github.com/hawtio/hawtio-next/blob/main/packages/hawtio/src/img/index.ts

Similarly, probably we should change the background images we use to conform to those of PF 5. The images currently used just come from PF 4 assets.
https://www.patternfly.org/components/login-page

Reply:
we'll work on the issue at another pull req.